src/main/java/com/sapred/ordermanagerred/controller/UserController.java

```diff
@@ -25,4 +28,34 @@ public ResponseEntity<String> logIn(@PathVariable("email") String email, @PathVa
     public void fill() {
         userService.fill();
     }
+
+    @PostMapping
+    public ResponseEntity addUser(@RequestBody User user) {
+        try {
+            User newUser = userService.addUser(user);
+            return ResponseEntity.ok().body(newUser);
+        }
+        catch (RuntimeException e) {
```

Comment:
no need to catch both RuntimeException  and Exception since RuntimeException  extends Exception 
please catch only Exception  and return INTERNAL_SERVER_ERROR